tools/cross-compile.sh

```diff
@@ -1,120 +1,107 @@
-cd ..
-
-mkdir -p binaries
-mkdir -p micro-$1
-
-cp LICENSE micro-$1
-cp README.md micro-$1
-cp LICENSE-THIRD-PARTY micro-$1
-cp assets/packaging/micro.1 micro-$1
-cp assets/packaging/micro.desktop micro-$1
-cp assets/micro-logo-mark.svg micro-$1/micro.svg
+#!/bin/sh
 
 HASH="$(git rev-parse --short HEAD)"
 VERSION="$(go run tools/build-version.go)"
 DATE="$(go run tools/build-date.go)"
 ADDITIONAL_GO_LINKER_FLAGS="$(go run tools/info-plist.go $VERSION)"
 
+mkdir -p binaries
+mkdir -p micro-$VERSION
+
+cp LICENSE micro-$VERSION
+cp README.md micro-$VERSION
+cp LICENSE-THIRD-PARTY micro-$VERSION
+cp assets/packaging/micro.1 micro-$VERSION
+cp assets/packaging/micro.desktop micro-$VERSION
+cp assets/micro-logo-mark.svg micro-$VERSION/micro.svg
+
+create_artefact_generic()
+{
+	mv micro micro-$VERSION/
+	tar -czf micro-$VERSION-$1.tgz micro-$VERSION
+	sha256sum micro-$VERSION-$1.tgz > micro-$VERSION-$1.tgz.sha
```

Comment:
nit: Just wondering, maybe it's better to use `.sha256` extension, to make it clear which sum it is?
What is the convention for sha256 sum file extension (if there is any)?